public/js/app.js

```diff
@@ -0,0 +1,165 @@
+        subImg.className = 'subImg';
+        subContent.appendChild(subImg);
+
+        let imgHere = document.createElement('img');
+        imgHere.className = 'redditImage';
+        imgHere.src = elem.data.thumbnail;
+        imgHere.onerror = function (){
+          imgHere.src = 'http://trifectaecosystems.com/wp-content/uploads/2015/04/snoo-300x208.jpg';
+        }
+        subImg.appendChild(imgHere);
+      }
+
+      let titleDiv = document.createElement('div');
+      titleDiv.className = 'title';
+      titleDiv.innerHTML = elem.data.title;
+      subContent.appendChild(titleDiv);
+
+      let statsDiv = document.createElement('div');
+      statsDiv.className = 'stats';
+      subContent.appendChild(statsDiv);
+
+      let author = document.createElement('div');
+      author.className = 'author statsItem';
+      author.innerHTML = elem.data.author;
+      statsDiv.appendChild(author);
+
+      let dot = document.createElement('div');
+      dot.className = 'statDot statsItem';
+      dot.innerHTML = '&#8226';
+      statsDiv.appendChild(dot);
+
+      function timeSince(date) {
+
+
+        let seconds = Math.floor((new Date() - date) / 1000);
+      
+        let interval = Math.floor(seconds / 31536000);
+      
+        if (interval > 1) {
+          return interval + " years";
+        }
+        interval = Math.floor(seconds / 2592000);
+        if (interval > 1) {
+          return interval + " months";
+        }
+        interval = Math.floor(seconds / 86400);
+        if (interval > 1) {
+          return interval + " days";
+        }
+        interval = Math.floor(seconds / 3600);
+        if (interval > 1) {
+          return interval + " hours";
+        }
+        interval = Math.floor(seconds / 60);
+        if (interval > 1) {
+          return interval + " minutes";
+        }
+        return Math.floor(seconds) + " seconds";
+      }
+
+      let timelapse = document.createElement('div');
+      timelapse.className = 'timelapse statsItem';
+      timelapse.innerHTML = timeSince(new Date(elem.data.created*1000))
+      statsDiv.appendChild(timelapse);
+
+      let dot2 = document.createElement('div');
+      dot2.className = 'statDot statsItem';
+      dot2.innerHTML = '&#8226';
+      statsDiv.appendChild(dot2);
+
+      let score = document.createElement('div');
+      score.className = 'score statsItem';
+      score.innerHTML = `${elem.data.score} upvotes`;
+      statsDiv.appendChild(score);
+
+      let selfText = document.createElement('div');
+      selfText.className = 'selfText';
+      selfText.innerHTML = elem.data.selftext;
+      subContent.appendChild(selfText);
+    });
+  }
+
+  let subRedditReq = new XMLHttpRequest();
+  subRedditReq.addEventListener('load', subRedditReqListener);
+  subRedditReq.open('GET', `https://www.reddit.com/r/${string}.json`);
+  subRedditReq.send();
+}
+
+logo.addEventListener('click', refreshWindow);
+
+function refreshWindow(){
+  location.reload();
```

Comment:
Instead of `location.reload()` which refresh the page and dumps all the currently running javascript, you could keep track of the current subreddit being viewed and just reload that content.